packages/auto-approve/src/check-pr.ts

```diff
@@ -15,8 +15,18 @@
 // eslint-disable-next-line node/no-extraneous-import
 import {ProbotOctokit} from 'probot';
 import {PullRequestEvent} from '@octokit/webhooks-definitions/schema';
-import {getChangedFiles} from './get-PR-info';
+import {getChangedFiles, File} from './get-PR-info';
```

Comment:
nit: I would use all lower case for file names:

* convention for JavaScript.
* also works better for case-insensitive file systems.

Reply:
File here is an interface I'm exporting (perhaps the name is too generic). I've noticed the convention for interfaces are uppercase; should I just change the name to something more specific?

Reply:
To assist, I think Ben meant the `'./get-PR-info'` part